cli/js/lib.deno.ns.d.ts

```diff
@@ -123,29 +123,39 @@ declare namespace Deno {
    */
   export function exit(code?: number): never;
 
-  /** Returns a snapshot of the environment variables at invocation. Changing a
-   * property in the object will set that variable in the environment for the
-   * process. The environment object will only accept `string`s as values.
-   *
-   *       const myEnv = Deno.env();
-   *       console.log(myEnv.SHELL);
-   *       myEnv.TEST_VAR = "HELLO";
-   *       const newEnv = Deno.env();
-   *       console.log(myEnv.TEST_VAR === newEnv.TEST_VAR);  // outputs "true"
-   *
-   * Requires `allow-env` permission. */
-  export function env(): {
-    [index: string]: string;
-  };
+  export const env: {
+    [key: string]: string | Function;
 
-  /** Retrieve the value of an environment variable. Returns undefined if that
-   * key doesn't exist.
-   *
-   *       console.log(Deno.env("HOME"));  // e.g. outputs "/home/alice"
-   *       console.log(Deno.env("MADE_UP_VAR"));  // outputs "Undefined"
-   *
-   * Requires `allow-env` permission. */
-  export function env(key: string): string | undefined;
+    /** Returns a snapshot of the environment variables at invocation. Changing a
+     * property in the object will set that variable in the environment for the
+     * process. The environment object will only accept `string`s as values.
+     *
+     *       const myEnv = Deno.env();
+     *       console.log(myEnv.SHELL);
+     *       myEnv.TEST_VAR = "HELLO";
+     *       const newEnv = Deno.env();
+     *       console.log(myEnv.TEST_VAR === newEnv.TEST_VAR);  // outputs "true"
+     *
+     * Requires `allow-env` permission. */
+    (): { [index: string]: string };
```

Comment:
This should be `entries()` or `getAll()` - I think the latter is better because `entries()` returns iterator and not and object @ry WDYT?